app/subscriber/src/features/press-gallery/PressGallery.tsx

```diff
@@ -24,27 +26,20 @@ export const PressGallery: React.FC = () => {
   const [results, setResults] = React.useState<any>([]);
   const [selected, setSelected] = React.useState<IContentModel[]>([]);
 
-  const defaultSettings = React.useMemo<IFilterSettingsModel>(() => {
-    return {
-      startDate: `${moment(filterAdvanced.startDate)}`,
-      endDate: `${moment(filterAdvanced.startDate).endOf('day')}`,
-      inByline: true,
-      inHeadline: true,
-      inStory: true,
-      searchUnpublished: false,
-      defaultOperator: 'or',
-    };
-  }, [filterAdvanced.startDate]);
-
-  const [pressSettings, setPressSettings] = React.useState<IFilterSettingsModel>(defaultSettings);
+  const [pressSettings, setPressSettings] = React.useState<IFilterSettingsModel>(
```

Comment:
might be nit picking here, but I actually prefer the name `defaultSettings` here.  If I was looking at this component, I would immediately make a connection as the "these are the default settings for the component" as opposed to `pressSettings` which could be "settings for pressing something"?  I try to follow the pattern of that `defaultSettings` name in components so that I know where I can find the things used "by default"

Reply:
I didn't rename the state variables.  I only replaced the const `defaultSettings` with a function `createFilterSettings`.